scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/SecurityContext.java

```diff
@@ -35,8 +35,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/", "/resources/**",
                        "/register").permitAll()
                 // The rest of the our application is protected.
-                .antMatchers("/users/**", "/admin/**").hasRole("ADMIN")
```

Comment:
I swapped the position of "USER" and "ADMIN" such that the new code was " .anyRequest().hasRole("ADMIN") ". I was having issues with the admin account not being able to access my views in /github/** until i did this.

Reply:
Now you just made everything only accessible for admins and nothing for users, `.antMatchers().hasRole("USER")` doesn't make much sense, it just doesn't give users access to anything. I would put any admin related actions under `/admin`, e.g. `/admin/github` and open that url only for admins.
`.antMatchers("/users/**", "/admin/**").hasRole("ADMIN")`.
basically says, any URL that starts with `/users` or `/admin` will be only accessible by users with the admin role. If you really want to keep `/github`, you can just add that to that line. However, it will get harder to keep track of which urls can be accessed by whom as your app grows. Prefixes help a lot with this.

`.anyRequest().hasRole("ADMIN")` says, everything that hasn't been mentioned above is only accessible by admins.